app/assets/javascripts/staff/events.js

```diff
@@ -7,5 +7,9 @@ $(document).ready(function () {
   $('.cancel-status-change').click(function() {
     $('.status-dropdown').hide();
     $('.btn-nav').show();
-  })
-});
\ No newline at end of file
+  });
+
+  $('.modal').on('keypress', function (e) {
+    if (e.which == 13) $(this).submit();
+  });
```

Comment:
I think I found it - there's a 'close' class on the modal buttons: <code>button.close data-dismiss="modal"</code>.. if I take the <code>.close</code> out we don't need the javascript workaround it seems.